### ISSUE TITLE
Warning: Not so important

### ISSUE THREAD

**Issue by Kalpesh89** [tensorlayer/srgan#143]
Code is working good. Only I am getting following warning at each step.
W0530 11:23:37.138493 140465058146112 optimizer_v2.py:928] Gradients does not exist for variables ['batchnorm_1/moving_mean:0', 'batchnorm_1/moving_var:0', 'batchnorm_2/moving_mean:0', 'batchnorm_2/moving_var:0', .......

I found that for some variables, we can't find gradient. So we need to tell them only trainable variables instead of all variables. But I can't find solution.

When I try to use tl.layer.get_variables_with_name, I am geting empty list. Can you solve this? Because due to so much warning, I can't figure out proper loss tracking for each step/epoch.

Or is there any solution to turn off this type of warning?
I tried -W ignore but it can't stop the message.

Thank you.

**Comment by zsdonghao** [comment]
thanks, it is fixed now

**Comment by 1LOVESJohnny** [comment]
Hi I met the same problem, do you know how to solve it?

**Comment by Kalpesh89** [comment]
I changed tensorlayer/tensorflow version(upgrade or degrade, exactly not
remembered). It solved for me.

**Comment by yangyingni** [comment]
> thanks, it is fixed now

Hello,can you tell me how did you fix it?Thank you very much.
I'm meeting this quetion:
```
Epoch: [1/100] step: [4/12] time: 8.715s, mse: 0.210 
WARNING:tensorflow:Gradients does not exist for variables ['batchnorm2d_1/moving_mean:0', 'batchnorm2d_1/moving_var:0', 
Epoch: [2/100] step: [0/12] time: 7.840s, mse: 0.233 
```
It can run,but shows Warning.
